aip/0122.md

```diff
@@ -18,18 +18,19 @@ resources.
 
 ## Guidance
 
-Most APIs reference resources only within their own scope. All resource names
-defined by an API **must** be unique within that API.
+All resource names defined by an API **must** be unique within that API. (See
```

Comment:
I've seen quite a few APIs which implicitly scope the resources by the requesting entity, effectively. I've generally tried to encourage them to use some hierarchy (e.g. users/ or projects/) to scope the resources properly, but haven't always succeeded in persuading the API producer to do this.

Am I reading correctly that those APIs would violate this AIP?